handler/OpenIDHandler.inc.php

```diff
@@ -92,9 +92,9 @@ function doAuthentication($args, $request, $provider = null)
 						[ROLE_ID_SITE_ADMIN, ROLE_ID_MANAGER, ROLE_ID_SUB_EDITOR, ROLE_ID_AUTHOR, ROLE_ID_REVIEWER, ROLE_ID_ASSISTANT],
 						$contextId
 					)) {
-						return $request->redirect($context, 'submissions');
+						return $request->redirect($context->getPath(), 'submissions');
```

Comment:
Does the existing code produce errors in your installation? I am testing it on stable-3_3_0 and I can't reproduce any problems.